backend/src/main/validation/PromotionValidation.ts

```diff
@@ -21,6 +21,9 @@ export class PromotionValidation {
     name: Joi.string().required(),
     description: Joi.string().required(),
     expirationDate: Joi.date().required(), // note incoming format is a string, and Joi will automatically convert to type Date
+    lat: Joi.number().required(),
+    lon: Joi.number().required(),
```

Comment:
Also should we enforce something like this? https://stackoverflow.com/questions/15965166/what-is-the-maximum-length-of-latitude-and-longitude

Reply:
Hmm.. Since we're always getting it from Google Places API I think it's okay?

Reply:
I think in general we may not want to design our endpoints for how our FE uses it. But rather define how our FE should use it. Probably the decimal places might be too much, but in the future say we no longer use Google Places API lat/lon values and we generate our own. This may be something to validate, because I think if our lat lon was only 1 decimal place, then that's only accurate to 11 km 😝

Reply:
LOOOL that's true!! I think we're def better off having more decimals than less, but yeah I think we can push it off for the future 😄